tests/Unit/Configuration/WildcardToRegexTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
Shuold we move this to Obfuscation namespace as well?

Reply:
Doh! 🤦‍♂ Nice catch! 😄